cli_tools/gce_ovf_import/main.go

```diff
@@ -153,7 +137,24 @@ func main() {
 		},
 	}
 
-	if err := service.RunWithServerLogging(service.InstanceImportAction, paramLog, project, runImport); err != nil {
+	params := buildImportParams()
```

Comment:
What's the motivation for making this function an anonymous function? (It's hard for me to trace what changed to fix your bug vs the refactoring)

Reply:
Because buildImportParams has to be executed before calling RunWithServerLogging. But I reverted this change in the latest commit by removing the ref to param.project in order to simplify the PR, so we can ignore this now.